kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaProtocolHandler.java

```diff
@@ -73,6 +73,8 @@
 
     private StatsLogger rootStatsLogger;
     private PrometheusMetricsProvider statsProvider;
+    private KoPBrokerLookupManager koPBrokerLookupManager;
```

Comment:
It's better to change the name to `KopBrokerLookupManager` and `kopBrokerLookupManager` to keep the name consistency with other classes.

Reply:
Ok